frontend/docker-compose.dev.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
```

Comment:
docker-compose up doesn't find  `frontend/docker-compose.dev.yml`, should I so something different?
![image](https://user-images.githubusercontent.com/6463001/115952060-bc1a2800-a4ec-11eb-91db-084d47583132.png)

Reply:
Added in README the instructions to run with other file, this case using docker-compose.dev.yml.